api/src/main/java/javax/enterprise/inject/spi/ProcessBeanAttributes.java

```diff
@@ -92,4 +92,16 @@
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public void veto();
+
+
+    /**
+     * Forces bean creation even if the bean class has non static and non private final methods.
```

Comment:
Something like: _"Instructs the container to ignore all non-static, final methods with public, protected or default visibility declared on any bean type of the specific bean during validation of injection points that require proxyable bean type."_ I know it's verbose but it's a spec anyway...